systemvm/debian/opt/cloud/bin/cs/CsAddress.py

```diff
@@ -627,11 +627,14 @@ def hasIP(self, ip):
         return ip in self.address.values()
 
     def arpPing(self):
-        cmd = "arping -c 1 -I %s -A -U -s %s %s" % (
-            self.dev, self.address['public_ip'], self.address['gateway'])
+        #bit of logging added inside the command itself, to make sure whether it's executed or not (due to possible race/timeout conditions
+        cmd = "arping -c 1 -I %s -A -U -s %s %s >> /tmp/arpping.log" % (self.dev, self.address['public_ip'], self.address['gateway'])
         if not self.cl.is_redundant() and (not self.address['gateway'] or self.address['gateway'] == "None"):
-            cmd = "arping -c 1 -I %s -A -U %s" % (self.dev, self.address['public_ip'])
+                cmd = "arping -c 1 -I %s -A -U %s >> /tmp/arpping1.log" % (self.dev, self.address['public_ip'])
```

Comment:
@andrijapanicsb maybe the intent is too far? maybe i am wrong... :hushed:

Reply:
@svenvogel this is just a Draft PR, won't get merged most probably - and I don't get your question?